src/parser.rs

```diff
@@ -1135,7 +1135,18 @@ impl<'run, 'src> Parser<'run, 'src> {
       token.get_or_insert(bracket);
 
       loop {
-        let name = self.parse_name()?;
+        let (name, inverted) = {
```

Comment:
I think this is probably cleaner than using mutable variables:

```suggestion
        let (name, inverted) = {
          let name = self.parse_name()?;
          if name.lexeme() == "not" {
            self.expect(ParenL)?;
            let name = self.parse_name()?;
            (name, true)
          } else {
            (name, false)
          }
        }
```

Reply:
And I think we should actually, since we don't have any invertable variables which aren't system variables, change the argument to `Attribute::new` to be !inverted, and within `Attribute::new` it's called `enabled`.